zerver/lib/generateData.py

```diff
@@ -0,0 +1,273 @@
+from __future__ import absolute_import
+from __future__ import print_function
+import itertools
+import json
+import random
+from typing import List, Dict, Any, Text, Optional
+from six.moves import range
+
+def loadConfig():
+    # type: () -> Dict [str, Any]
+    infile = open("zerver/lib/config.generate_data.json", "r")
+    config = json.loads(infile.read())
+
+    return config
+
+
+def getStreamTitle(gens):
+    # type: (Dict[str, Any]) -> str
+
+    # the secret to generating unique data is:
+    # make sure that count of each list is a different prime number
+    # with these four, we will get 15015 unique values before a dupe
+    # 15 * 11 * 7 * 5 * 3 = 15015
```

Comment:
I would remove (most of) this comment; for one thing, it's not actually important that the numbers be prime :)